### PR TITLE
Chore: adds snippets for get started section for 6.0

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request the snippets required for the get started section are included and the markdown files are now adjusted to reflect the changes:

What was done:
- Updated the markdown files to include the snippets based on the file naming and structure decided.
- Created the snippets (react,react ts, angular)

Feel free to provide feedback

### CODE REVIEW

File: docs/get-started/setup.md [hunk 1]
```diff
@@ -4,25 +4,16 @@ title: 'Setup'
...
 
-```js
-// YourComponent.stories.js
+<!-- prettier-ignore-start -->
 
-import { YourComponent } from './YourComponent';
+<CodeSnippets
+  paths={[
+    'common/your-component.js.mdx',
+    'common/your-component.ts.mdx',
```
Comment (tmeasday) [inline docs/get-started/setup.md:12]: Isn't this snippet react-specific? (has rendering of the template)

File: docs/get-started/setup.md [hunk 2]
```diff
@@ -60,15 +51,15 @@ A common frontend pattern is for components to assume that they render in a cert
...
-// .storybook/preview.js
+<!-- prettier-ignore-start -->
 
-import { ThemeProvider } from 'styled-components';
-export const decorators = [
-  (Story) => <ThemeProvider theme="default"><Story/></ThemeProvider>,
-];
+<CodeSnippets
+  paths={[
+    'common/storybook-preview-with-styled-components-decorator.js.mdx',
```
Comment (tmeasday) [inline docs/get-started/setup.md:58]: This is definitely react-specific!

File: docs/get-started/setup.md (hunk 1 above)
Comment (jonniebigodes) [reply docs/get-started/setup.md:12]: the ts file is actually extracted from the angular, i was aiming for a "dummy example" for both js and ts options. Didn't want to go into specifics. If need be i'll adjust the examples to become react /react with ts and for angular. 

Let me know.

File: docs/get-started/setup.md (hunk 2 above)
Comment (jonniebigodes) [reply docs/get-started/setup.md]: the snippet is now in the correct place and the page is updated. Thanks for catching that

**Pull request merged**